the-blue-alliance-ios/View Controllers/Teams/TeamsViewController.swift

```diff
@@ -184,6 +190,10 @@ class TeamsViewController: TBATableViewController, Refreshable, Stateful, TeamsV
     // MARK: TableViewDataSourceDelegate
 
     func controllerDidChangeContent() {
+        // Don't update our interface if we're in the background - since updating the search bar count is expensive
+        guard let nav = navigationController, nav.visibleViewController == parent else {
+            return
+        }
         DispatchQueue.main.async { [weak self] in
             self?.updateInterface()
```

Comment:
This is still super slow. I think we can set the batch size and make this a little faster?